src/workerd/jsg/setup.c++

```diff
@@ -151,6 +151,11 @@ void V8System::init(kj::Own<v8::Platform> platformParam,
   // more flags.)
   v8::V8::SetFlagsFromString("--noincremental-marking");
 
+  // These features are completed and enabled by default in Chrome, but not
+  // in V8. Follows Node.js: https://github.com/nodejs/node/pull/58154
+  v8::V8::SetFlagsFromString("--js-explicit-resource-management");
+  v8::V8::SetFlagsFromString("--js-float16array");
```

Comment:
It would probably be helpful to add a couple of simple tests that verify the flags are properly set. I know the compression-test covers the `Float16Array` usage but we should also have one that covers ERM

Reply:
Any recommendations on what to test?

Reply:
I'll do a follow up for js-explicit-resource-management tests.

Reply:
Simple test:

```js
import { strictEqual } from 'node:assert';
import { mock } from 'node:test';

// Test verifies that explicit resource management is, in fact, enabled.

export const disposeTest = {
  test() {
    const fn = mock.fn();
    {
      using _ = {
        [Symbol.dispose]() {
          fn();
        },
      };
    }
    strictEqual(fn.mock.callCount(), 1);
  },
};

export const asyncDisposeTest = {
  async test() {
    const fn = mock.fn(async () => {});
    {
      await using _ = {
        async [Symbol.asyncDispose]() {
          await fn();
        },
      };
    }
    strictEqual(fn.mock.callCount(), 1);
  },
};

```